src/lib.rs

```diff
@@ -93,6 +97,15 @@
 //!             0x14, /* b <this address>; shouldn't get here, but if so loop forever */
 //!         ];
 //!     }
+//!     #[cfg(target_arch = "riscv64")]
```

Comment:
Ah, I see now. I guess here in this top level documentation, having something for each supported arch is fine :)